bootstrap/README.md

```diff
@@ -2,8 +2,10 @@
 
 To use this existing project structure and scripts for your new ML project, you can quickly get started from the existing repository,  bootstrap and create a template that works for your ML project. Bootstraping will prepare a similar directory structure for your project which includes renaming files and folders, deleting and cleaning up some directories and fixing imports and absolute path based on your project name. This will enable reusing various resources like pre-built pipelines and scripts for your new project.
 
-To bootstrap from the existing MLOpsPython repository clone this repository and run bootstrap.py script as below
+To bootstrap from the existing MLOpsPython repository clone this repository, ensure Python is installed locally, and run bootstrap.py script as below
```

Comment:
Python 3.x?

Reply:
we're only using `os, sys, argparse` which should technically work on 2.x or 3.x